src/NHSD.BuyingCatalogue.Ordering.Domain/ServiceRecipient.cs

```diff
@@ -5,11 +5,15 @@ namespace NHSD.BuyingCatalogue.Ordering.Domain
 {
     public sealed class ServiceRecipient
     {
+        [Key]
```

Comment:
Are these attributes still necessary since the mapping is done in the entity type configuration?